LoopKitUI/View Controllers/DismissableHostingController.swift

```diff
@@ -21,11 +20,12 @@ public class DismissibleHostingController: UIHostingController<AnyView> {
         rootView: Content,
         dismissalMode: DismissalMode = .modalDismiss,
         isModalInPresentation: Bool = true,
-        onDisappear: @escaping () -> Void = {}
+        onDisappear: @escaping () -> Void = {},
+        colorPalette: ColorPalette? = nil
```

Comment:
Actually, this is a little tricky.. it means everywhere we instantiate one of these hosting controllers, the real palette will have to be accessible, which means we can't instantiate one from within LoopKit. What do you think about making a public static variable that we assign once from Loop when the app launches? I wouldn't normally advocate for global state, but I don't think we have a lot of options.